main.tf

```diff
@@ -77,6 +77,7 @@ resource "kubernetes_deployment" "default" {
               read_only         = lookup(csi_secret_volume.value, "read_only", null)
               driver            = lookup(csi_secret_volume.value, "driver", null)
               volume_attributes = lookup(csi_secret_volume.value, "volume_attributes", null)
+              volume_handle     = lookup(csi_secret_volume.value, "volume_handle", null)
```

Comment:
Why is this added? Don't touch this unless you have validated that secrets mounting still works correctly.

Reply:
I ran terraform validate and it showed me an error that volume_handle is missing

Reply:
There is an issue with the provider version then. That param has a history of problems and shouldn't be added.

Reply:
In providers we are using an old version of kubernetes provider

Reply:
Upgrade to the latest version and let's see what happens. Coordinate with @tsaucier-sf on testing.

Reply:
sure